code/game/turfs/open.dm

```diff
@@ -145,7 +145,7 @@
 			C.slipping = TRUE
```

Comment:
Setting slipping to TRUE here does nothing, because it is immediately set to FALSE after.